doc/schemas/WRITING_SCHEMAS.md

```diff
@@ -10,6 +10,13 @@ use a subset of the full [JSON Schema Specification](https://json-schema.org/),
 but if you find that limiting it's probably a sign that you should simplify
 your JSON output.
 
+## Updating a Schema
+
+If you add a field, you should add it to the schema, and you must add
+"added": "VERSION" (where VERSION is the next release version!).
+
+Similarly, if you deprecate a field, add "deprecated": "VERSION".
```

Comment:
Is this meant to be the version it was deprecated, or the version that we aim to remove the field? Or are we removing the field definition along with the deprecation directly from the schema once we enforce the deprecation?

Reply:
Version it was deprecated, and yes, it gets removed.  But this does mean the doc tool has to calc when it will be removed...

Reply:
Added this explicitly to that document...